makes/src/expat

```
@@ -0,0 +1,12 @@
+#!/bin/zsh
```

Is this intended to be zsh?

---

I believe that was simply a artifact from the windows makes. shellcheck appears to be happy with /bin/sh so I will replace it and wait for the CI to complete.

---

I think you missed a couple.

---

Seems like expat silently used a zsh feature that looked like a bash feature, should *hopefully* be fixed now.

As for the other files I left everything the same except for in the makes folder